src/dftbp/common/envcheck.c

```diff
@@ -0,0 +1,27 @@
+/*------------------------------------------------------------------------------------------------*/
+/*  DFTB+: general package for performing fast atomistic simulations                              */
+/*  Copyright (C) 2006 - 2022  DFTB+ developers group                                             */
+/*                                                                                                */
+/*  See the LICENSE file for terms of usage and distribution.                                     */
+/*------------------------------------------------------------------------------------------------*/
+
+#include <sys/resource.h>
```

Comment:
Might have a `portability` issue](https://www.gnu.org/software/gnulib/manual/html_node/sys_002fresource_002eh.html)

Reply:
That's a pity, does it affect any of our supported/tested build environments?

Reply:
No, but the BSD packaging guy will complain if we don't include the other two sys headers.

Reply:
I have no problem with adding the additional headers.

Reply:
If, we would only need `sys/types.h` on BSD. But, I'd leave it for the moment as it is, it probably also works on BSD.